Tools/Cli-LoRa-Device-Provisioning/DefaultRouterConfig/EU863.json

```diff
@@ -1,121 +1,123 @@
+        "FFFFFFFFFFFFFFFF"
+      ]
     ],
-    [
-      10,
-      125,
-      0
+    "region": "EU863",
+    "hwspec": "sx1301/1",
+    "freq_range": [
+      863000000,
+      870000000
     ],
-    [
-      9,
-      125,
-      0
+    "DRs": [
+      [
+        12,
+        125,
+        0
+      ],
+      [
+        11,
+        125,
+        0
+      ],
+      [
+        10,
+        125,
+        0
+      ],
+      [
+        9,
+        125,
+        0
+      ],
+      [
+        8,
+        125,
+        0
+      ],
+      [
+        7,
+        125,
+        0
+      ],
+      [
+        7,
+        250,
+        0
+      ]
     ],
-    [
-      8,
-      125,
-      0
-    ],
-    [
-      7,
-      125,
-      0
-    ],
-    [
-      7,
-      250,
-      0
-    ]
-  ],
-  "sx1301_conf": [
-    {
-      "radio_0": {
-        "enable": true,
-        "freq": 867500000
-      },
-      "radio_1": {
-        "enable": true,
-        "freq": 868500000
-      },
-      "chan_FSK": {
-        "enable": true,
-        "radio": 1,
-        "if": 300000
-      },
-      "chan_Lora_std": {
-        "enable": true,
-        "radio": 1,
-        "if": -200000,
-        "bandwidth": 250000,
-        "spread_factor": 7
-      },
-      "chan_multiSF_0": {
-        "enable": true,
-        "radio": 1,
-        "if": -400000
-      },
-      "chan_multiSF_1": {
-        "enable": true,
-        "radio": 1,
-        "if": -200000
-      },
-      "chan_multiSF_2": {
-        "enable": true,
-        "radio": 1,
-        "if": 0
-      },
-      "chan_multiSF_3": {
-        "enable": true,
-        "radio": 0,
-        "if": -400000
-      },
-      "chan_multiSF_4": {
-        "enable": true,
-        "radio": 0,
-        "if": -200000
-      },
-      "chan_multiSF_5": {
-        "enable": true,
-        "radio": 0,
-        "if": 0
-      },
-      "chan_multiSF_6": {
-        "enable": true,
-        "radio": 0,
-        "if": 200000
-      },
-      "chan_multiSF_7": {
-        "enable": true,
-        "radio": 0,
-        "if": 400000
+    "sx1301_conf": [
+      {
+        "radio_0": {
+          "enable": true,
+          "freq": 867500000
+        },
+        "radio_1": {
+          "enable": true,
+          "freq": 868500000
+        },
+        "chan_FSK": {
+          "enable": true,
+          "radio": 1,
+          "if": 300000
+        },
+        "chan_Lora_std": {
+          "enable": true,
+          "radio": 1,
+          "if": -200000,
+          "bandwidth": 250000,
+          "spread_factor": 7
+        },
+        "chan_multiSF_0": {
+          "enable": true,
+          "radio": 1,
+          "if": -400000
+        },
+        "chan_multiSF_1": {
+          "enable": true,
+          "radio": 1,
+          "if": -200000
+        },
+        "chan_multiSF_2": {
+          "enable": true,
+          "radio": 1,
+          "if": 0
+        },
+        "chan_multiSF_3": {
+          "enable": true,
+          "radio": 1,
```

Comment:
Are you sure about this change?

Original EU863 [global.conf](https://github.com/Lora-net/packet_forwarder/blob/master/lora_pkt_fwd/cfg/global_conf.json.PCB_E286.EU868.basic) had this channel on radio 0

Reply:
Nope that was a typo I will revert

Reply:
Done, thank you very much for the catch, I might have copied these in the docs, I will also revert in case.